src/core/directives/google-map-data-layer.ts

```diff
@@ -0,0 +1,67 @@
+import {Directive, EventEmitter, OnDestroy, OnInit} from '@angular/core';
+import {Subscription} from 'rxjs/Subscription';
+
+import {DataMouseEvent} from './../services/google-maps-types';
+import {DataLayerManager} from './../services/managers/data-layer-manager';
+
+let layerId = 0;
+
+@Directive({
```

Comment:
Can you add a description with an example? Please check out the other directives for examples